easybuild/easyconfigs/o/OpenMM/OpenMM-6.3-foss-2015a.eb

```diff
@@ -0,0 +1,26 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Author:    Stephane Thiell <sthiell@stanford.edu>
+##
+name = 'OpenMM'
+version = '6.3'
+
+homepage = 'http://openmm.org/'
+description = "A high performance toolkit for molecular simulation."
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+source_urls = ['https://github.com/pandegroup/openmm/releases']
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [
+    ('CMake', '3.2.3'),
+    ('libxml2', '2.9.2'),
+    ('SWIG', '3.0.8', '-Python-2.7.11'),
+    ('Doxygen', '1.8.7'),
+]
+
+dependencies = [('Python', '2.7.11')]
```

Comment:
@thiell the easyblock has a hard check for CUDA, but it is not listed as a dependency? So how can this have worked?